README.md

```diff
@@ -49,17 +49,17 @@ dotnet add package Jellyfin.Model
 dotnet add package Jellyfin.Controller
 ```
 
-You have an autogenerated Class1.cs file, you won't be needing this, so go ahead and delete it.
+You have an autogenerated Class1.cs file. You won't be needing this, so go ahead and delete it.
 
-## 2. Setup Basics
+## 2. Set Up the Basics
 
 There are a few mandatory classes you'll need for a plugin so we need to make them.
 
-### Make a new class called PluginConfiguration
+### PluginConfiguration
 
 You can call it whatever you'd like really. This class is used to hold settings your plugin might need. We can leave it empty for now. This class should inherit from `MediaBrowser.Model.Plugins.BasePluginConfiguration`
 
-### Make a new class called Plugin
+### Plugin
 
 This is the main class for your plugin. It will define your name, version and Id. It should inherit from `MediaBrowser.Common.Plugins.BasePlugin<PluginConfiguration> `
```

Comment:
```suggestion
This is the main class for your plugin. It will define your name, version and Id. It should inherit from `MediaBrowser.Common.Plugins.BasePlugin<PluginConfiguration>`
```